src/edu/csus/ecs/pc2/clics/API202306/AwardService.java

```diff
@@ -0,0 +1,260 @@
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Singleton;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.PATCH;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Feature;
+import javax.ws.rs.core.FeatureContext;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.ext.Provider;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import edu.csus.ecs.pc2.core.IInternalController;
+import edu.csus.ecs.pc2.core.imports.clics.CLICSAward;
+import edu.csus.ecs.pc2.core.imports.clics.CLICSAwardUtilities;
+import edu.csus.ecs.pc2.core.log.Log;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+import edu.csus.ecs.pc2.services.core.JSONUtilities;
+import edu.csus.ecs.pc2.services.eventFeed.WebServer;
+
+/**
+ * WebService to handle awards endpoint
+ * @author John Buck
+ *
+ */
+@Path("/contests/{contestId}/awards")
+@Produces(MediaType.APPLICATION_JSON)
+@Provider
+@Singleton
+public class AwardService implements Feature {
+
+    private IInternalContest model;
+    @SuppressWarnings("unused")
+    private IInternalController controller;
+
+    public AwardService(IInternalContest inContest, IInternalController inController) {
+        super();
+        this.model = inContest;
+        this.controller = inController;
+    }
+
+    /**
+     * This method returns a representation of the current contest awards in JSON format. 
+     * The response is a JSON array with one award description per array element, complying with 2023-06
+     * 
+     * @param sc Security info for user making request
+     * @param contestId The contest id
+     * @return a {@link Response} object containing the groups in JSON form
```

Comment:
I think this is supposed to say "containing the AWARDS in json form"?

Reply:
Yep.  Will fix.

Reply:
Fixed in next push.